cli/src/find_path.rs

```diff
@@ -0,0 +1,46 @@
+use super::pager::Pager;
+use asset_identifier;
+use clap::ArgMatches;
+use error::Result;
+use fmt::{Formatter, Simple};
+use stellar_client::{sync, endpoint::payment, resources::Amount, sync::Client};
+
+pub fn find_path(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+
+    let endpoint = {
+        let destination_account = matches
+            .value_of("destination_account")
+            .expect("Destination account is a required field");
+        let source_account = matches
+            .value_of("source_account")
+            .expect("Source account is a required field");
+        let destination_amount = matches
+            .value_of("destination_amount")
+            .expect("Destination amount is a required field");
+        let destination_asset_str = matches
+            .value_of("destination_asset")
+            .expect("Destination asset is a required field");
+        let destination_asset = asset_identifier::from_str(destination_asset_str)?;
+        let endpoint = payment::FindPath::new(
+            source_account,
+            destination_account,
+            destination_asset,
+            Amount::new(destination_amount.parse::<i64>().unwrap() * 10_000_000),
```

Comment:
Parsing to an `i64` will lose the decimals: `1.123` will be `1`, etc. We should try to capture what they mean exactly, similar to how we do it in the client library. In fact, it might make sense to even implement a `FromStr` trait on the amount type so that we can handle the parsing cleanly (and without fidelity loss).

Reply:
Ok, so this was fun, but a little wonky -- I noted in the code w/ comments where the wonkiness comes into play.